backend/server.js

```diff
@@ -8,6 +8,7 @@ const { ErrorHandler, handleError } = require("./middleware/errorMiddleware.js")
 const { connectDB } = require("./config/db.js")
 
 const userRoutes = require("./routes/userRoutes.js")
+const passRoutes = require("./routes/passRoutes.js")
 
 dotenv.config()
```

Comment:
Is this required here? Isn't we have done this in server.js?

Reply:
this is server.js itself na, sir?